docs/contribute/colima.qmd

```diff
@@ -0,0 +1,130 @@
+# Colima setup for MacOS M1/M2 users
+
+If you are working with an `arm64` architecture (Mac M1/M2), you will need to
+run docker to  be able to test some backends supported by Ibis. With Colima,
+you can run docker in Mac m1/m2 without Docker Desktop.
+
+## Colima
+
+[Colima](https://github.com/abiosoft/colima) is container runtimes on macOS
+(and Linux) with minimal setup, that has M1 macs support.
+
+### Get started
+
+1. Uninstall Docker Desktop
+
+If you have Docker desktop installed, followed the references, both of them have
+a section on how to get rid of it.
+
+2. Install Docker Client with `brew`
+
+Make sure you brew is up to date and cashed cleaned up, just in case.
+
+```bash
+$ brew install docker
+```
+Check installation by doing
+
+```bash
+$ docker --version
+```
+
+```bash
+$ docker version
```

Comment:
Are these two commands meaningfully different? Do we need to write both?

Reply:
They are, the second won't run because there is no engine installed yet. But I can rework this to make it more clear.